openbb_terminal/cryptocurrency/defi/defipulse_model.py

```diff
@@ -12,6 +12,23 @@
 logger = logging.getLogger(__name__)
 
 
+@log_start_end(log=logger)
+def num_to_int(num):
+    if num[-1] == "M":
+        return int(float(num[1:-1])) * 1000000
```

Comment:
For visibility, you can do: `1_000_000` which is quite cool.

Also, int(float(X)) is redundant, you can just do `int`